tests/system/action/user/test_create.py

```diff
@@ -147,7 +147,7 @@ def test_create_template_fields(self) -> None:
         assert user.get("vote_weight_$1") == "1.000000"
         assert user.get("vote_weight_$2") == "2.333333"
         assert set(user.get("vote_weight_$", [])) == {"1", "2"}
-        assert user.get("meeting_ids") == [1, 2]
+        assert set(user.get("meeting_ids", [])) == {1, 2}
```

Comment:
To check lists IMO the best is self.assertCountEqual(list1, list2), which compares lists without consideration of sequence. `set` changes lists by removing dublicates

Reply:
Ah, good call, I did not know about this method! Will change